packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -502,6 +503,8 @@ export interface AnalyticalTablePropTypes extends Omit<CommonProps, 'title'> {
   tableInstance?: Ref<Record<string, any>>;
 }
 
+const uniqueId = getRandomId();
```

Comment:
I think this could cause issues when multiple table instances are on the same page.
I would prefer to use our `useIsomorphicId` hook instead.